vgen/src/test/java/io/gapi/vgen/testdata/python_no_path_templates.baseline

```diff
@@ -98,18 +89,20 @@ class NoTemplatesServiceApi(object):
         """
         if scopes is None:
             scopes = self._ALL_SCOPES
-        bundling_override = bundling_override or {}
-        retrying_override = retrying_override or {}
-        client_config = pkg_resources.resource_string(
-            __name__, 'no_templates_service_client_config.json')
-        self._defaults = api_callable.construct_settings(
-            'google.example.v1.NoTemplatesService',
-            json.loads(client_config),
-            bundling_override,
-            retrying_override,
-            config.STATUS_CODE_NAMES,
-            timeout
-        )
+        if isinstance(client_config, google.gax.CallSettings):
+            self._defaults = client_config
```

Comment:
Ow, right.
Then probably call_config parameter should be always dict() (Hash in Ruby), which is usually the return value of construct_settings, instead of raw JSON values.  Updated to do so -- both code and the comment.